persistence-modules/hibernate-annotations/src/main/java/com/baeldung/hibernate/customtypes/OfficeEmployee.java

```diff
@@ -34,7 +34,7 @@ public class OfficeEmployee {
             @AttributeOverride(name = "cityCode", column = @Column(name = "city_code")),
             @AttributeOverride(name = "number", column = @Column(name = "number"))
     })
-    @Type(value = PhoneNumberType.class)
+    @CompositeType(value = PhoneNumberType.class)
```

Comment:
the employeeNumber remains a composite type with multiple columns in the database